src/AbstractOperations/at.jl

```diff
@@ -93,7 +93,7 @@ function interpolate_index(a::UnitRange, b::UnitRange, loc, new_loc)
     # Abstract operations that require an interpolation of a sliced fields are not supported!
     first(a) > last(a) && throw(ArgumentError("Cannot interpolate a Sliced field from $loc to $(new_loc)!"))
```

Comment:
```suggestion
    first(a) > last(a) && throw(ArgumentError("Cannot interpolate a SlicedField from $loc to $(new_loc)!"))
```

or just "sliced field" if we are not planning to define the type alias SlicedField (we reserve TitleCase for types + type aliases)